### PR TITLE
[#208] Get Docker working and update README

### PR DESCRIPTION
**Pull request by noahko96**
### Addresses #208 

- Gets Docker working
- Updates `README` to actually be halfway decent.

### CODE REVIEW

**Review by solomonhawk — APPROVED** [review]
LGTM 🙌

**Comment by dce** [comment]
I'll pull this down onto my m1 and see if it builds. This line (in `Dockerfile`) looks like it might be an issue:

```
wget -q --continue -P $CHROMEDRIVER_DIR "http://chromedriver.storage.googleapis.com/$CHROMEDRIVER_VERSION/chromedriver_linux64.zip" && \
```

**Comment by noahko96** [comment]
> I'll pull this down onto my m1 and see if it builds. 

@dce I just wanted to see if you ever got the chance to do this.

File: docker-compose.yml [hunk 1]
```diff
@@ -1,38 +1,67 @@
...
-  log:
-  tmp:
-
 services:
   db:
-    environment:
-      POSTGRES_USER: root
-      POSTGRES_DB: adventure-time_development
-    image: postgres:9.3-alpine
+    image: postgres:13.3-alpine
```
[the remark this answers is not in the document]
Comment (noahko96) [reply docker-compose.yml]: It does not match the deployed version, so I swapped it down to 10.6 so it would.

**Comment by noahko96** [comment]
I am just going to merge this in for now. If it does not work on M1s, I will fix that later. This still feels like a step in the right direction either way since it previously didn't work on any type of computer.

**Comment by dce** [comment]
Yeah, sorry, meant to pull this down, just didn't get to it.

**Pull request merged**